org.gnome.gThumb.yml

```
@@ -0,0 +1,81 @@
+app-id: org.gnome.gThumb
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+
+command: gthumb
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-public-share
+  - --filesystem=xdg-pictures
+
+
+modules:
+  - name: exiv2
+    buildsystem: cmake-ninja
+    cleanup:
+      - /lib/exiv2
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DEXIV2_BUILD_EXIV2_COMMAND=OFF
+      - -DEXIV2_BUILD_SAMPLES=OFF
+    sources:
+      - type: archive
+        url: https://exiv2.org/builds/exiv2-0.27.4-Source.tar.gz
+        sha256: 84366dba7c162af9a7603bcd6c16f40fe0e9af294ba2fd2f66ffffb9fbec904e
+
+  - name: lcms2
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/lcms/files/lcms/2.12/lcms2-2.12.tar.gz
+        sha256: 18663985e864100455ac3e507625c438c3710354d85e5cbb7cd4043e11fe10f5
+
+  # intltool needed by colord 1.3.5
+  - shared-modules/intltool/intltool-0.51.json
+  # colord 1.3.5 can be built without gusb and its dependencies, while 1.4.x require gusb.
+  - name: colord
+    config-opts:
+      - --disable-gusb
+      - --disable-udev
+      - --disable-polkit
+      - --disable-systemd-login
+      - --disable-argyllcms-sensor
+      - --disable-bash-completion
+      - --with-tmpfilesdir=/app/lib
+      - --with-systemdsystemunitdir=/app/lib
+      - --with-udevrulesdir=/app/lib
+    sources:
+      - type: archive
+        url: https://www.freedesktop.org/software/colord/releases/colord-1.3.5.tar.xz
+        sha256: 2daa8ffd2a532d7094927cd1a4af595b8310cea66f7707edcf6ab743460feed2
+
+  - name: libraw
+    buildsystem: simple
+    config-opts:
+      - --disable-examples
+    build-commands:
+      - autoreconf --install --force
+      - ./configure --prefix=/app
+      - make
+      - make install
+    sources:
+      - type: archive
+        url: https://www.libraw.org/data/LibRaw-0.20.2.tar.gz
+        sha256: dc1b486c2003435733043e4e05273477326e51c3ea554c6864a4eafaff1004a6
```

```suggestion
  - name: libraw
    config-opts:
      - --disable-examples
    sources:
      - type: archive
        url: https://www.libraw.org/data/LibRaw-0.20.2.tar.gz
        sha256: dc1b486c2003435733043e4e05273477326e51c3ea554c6864a4eafaff1004a6
      - type: shell
        commands:
        - autoreconf -fi
```

With this it works. I'm not sure what the problem was your were having, and why `--force --install` didn't seem to do it.

---

You are right, it works